portsyncd/portsyncd.cpp

```diff
@@ -134,6 +135,13 @@ int main(int argc, char **argv)
                     p.set("PortInitDone", attrs);
                     SWSS_LOG_NOTICE("PortInitDone");
 
+                    /*
+                     * Notify "System ready" feature that we are ok.
+                     */
+                    FieldValueTuple app_state("up_status", "true");
```

Comment:
> Why is portinitdone alone considered for sysready? In case of warmboot/fastboot scenario should we consider reconciliation done as sys ready?

@dgsudharsan we don't have other requirements and portinitdone is good enough. The previous design used "boot-up" of the service as "system ready".
I am not sure about "reconciliation done". Is warmboot/fastbot not set "PortInitDone"?